cluster-autoscaler/core/autoscaler.go

```diff
@@ -50,9 +50,9 @@ type Autoscaler interface {
 
 // NewAutoscaler creates an autoscaler of an appropriate type according to the parameters
 func NewAutoscaler(opts AutoscalerOptions, predicateChecker *simulator.PredicateChecker, kubeClient kube_client.Interface,
-	kubeEventRecorder kube_record.EventRecorder, listerRegistry kube_util.ListerRegistry) (Autoscaler, errors.AutoscalerError) {
+	kubeEventRecorder kube_record.EventRecorder, listerRegistry kube_util.ListerRegistry, providerStopChannel chan struct{}) (Autoscaler, errors.AutoscalerError) {
 
-	autoscalerBuilder := NewAutoscalerBuilder(opts.AutoscalingOptions, predicateChecker, kubeClient, kubeEventRecorder, listerRegistry)
+	autoscalerBuilder := NewAutoscalerBuilder(opts.AutoscalingOptions, predicateChecker, kubeClient, kubeEventRecorder, listerRegistry, providerStopChannel)
```

Comment:
this changes are just to push through the stop channel, which we don't seem to be using anyway